### PR TITLE
Переделываем Локализацию

### PR DESCRIPTION
**Pull request by Pasha8914**
# Pull request template

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/Contribution) guide
- [x] branch rebased on `develop`
- [x] styleguide check `npm run validate`
- [x] good naming
- [ ] keep simple
- [ ] video or screenshot attached
- [x] testing instruction attached
- [x] check your PR once again

### CODE REVIEW

File: shared/components/modals/WithdrawModal/WithdrawModal.js [hunk 1]
```diff
@@ -113,116 +115,136 @@ export default class WithdrawModal extends React.Component {
...
+            )
+          }
+          <FieldLabel inRow>
+            <FormattedMessage id="Withdrow1194" defaultMessage="Address " />
+            {' '}
+            <Tooltip id="WtH203" >
+              <div style={{ textAlign: 'center' }}>
+                <FormattedMessage
+                  id="WTH275"
+                  defaultMessage="Make sure the wallet you {br }are sending the funds to supports {currency}"
```
Comment (sdwas) [inline shared/components/modals/WithdrawModal/WithdrawModal.js]:
```suggestion
                  defaultMessage="Make sure the wallet you {br}are sending the funds to supports {currency}"
```

Comment (noxonsu) [reply shared/components/modals/WithdrawModal/WithdrawModal.js]: травис походу будет орать

File: shared/pages/Swap/BtcToEth.js [hunk 2]
```diff
@@ -269,7 +264,8 @@ export default class BtcToEth extends Component {
                           >
                             <FormattedMessage id="BtcToEth233" defaultMessage="How refund your money ?" />
                           </a>
-                          Refund hex transaction: <code> {flow.refundTxHex}</code>
+                          <FormattedMessage id="BtcToEth248" defaultMessage="Refund hex transaction: " />
+                          <code> {flow.refundTxHex}</code>
```
Comment (sdwas) [inline shared/pages/Swap/BtcToEth.js]:
```suggestion
                          <code> {flow.refundTxHex} </code>
```

File: shared/pages/History/History.js [hunk 3]
```diff
@@ -57,14 +65,19 @@ export default class History extends Component {
   )
 
   render() {
-    const { items, swapHistory } = this.props
-    const titles = [ 'Coin', 'Status', 'Statement', 'Amount' ]
+    const { items, swapHistory, formatMessage } = this.props
```
Comment (sdwas) [inline shared/pages/History/History.js]:
```suggestion
    const { items, swapHistory, formatMessage, intl } = this.props
```

File: shared/pages/History/History.js [hunk 4]
```diff
@@ -57,14 +65,19 @@ export default class History extends Component {
...
+      <FormattedMessage id="Coin61" defaultMessage="Coin" />,
+      <FormattedMessage id="Status61" defaultMessage="Status" />,
+      <FormattedMessage id="Statement61" defaultMessage="Statement" />,
+      <FormattedMessage id="Amount61" defaultMessage="Amount" />,
+    ]
 
     return (
       <section>
-        <PageHeadline subTitle="History" />
+        <PageHeadline subTitle={formatMessage(subTitle.subTitleHistory)} />
```
Comment (sdwas) [inline shared/pages/History/History.js]: Fix history page:
```suggestion
        <PageHeadline subTitle={intl.formatMessage(subTitle.subTitleHistory)} />
```

**Comment by noxonsu** [comment]
https://airdb.wpmix.net/chrome_2018-12-16_00-03-0627ab9dfa-d9ce-470b-9c80-a97d9ba3d716.png тут должно быть ru

**Pull request merged**